common/kernel/command.cc

```diff
@@ -189,6 +189,7 @@ po::options_description CommandHandler::getGeneralOptions()
     general.add_options()("placer-heap-critexp", po::value<int>(),
                           "placer heap criticality exponent (int, default: 2)");
     general.add_options()("placer-heap-timingweight", po::value<int>(), "placer heap timing weight (int, default: 10)");
+    general.add_options()("no-placer-timeout", "allow the placer to attempt placement without timeout");
```

Comment:
I would suggest to reflect the usage options text to users more clearly:
add_options()("placer-timeout", "....placer timeout.....,  XXXXX seconds default, -1 means disabled")

Feel free to do other default/no-default/counter/seconds variations.